examples/amp-story/amp-story-shopping.html

```diff
@@ -17,21 +17,65 @@
+          This is an example of a JSON config retrieved from a source file.
+          To use the inline JSON config, remove the src attribute.
+        -->
+        <amp-story-shopping-config layout='nodisplay' src="/examples/amp-story/shopping/remote.json" >
+          <script type="application/json">
+            {
+                "items" : [
+                   {
+                    "product-tag-id": "hat",
+                    "brand-label": "...",
+                    "brand-favicon": "...",
+                    "product-title": "Hootenanny Hat",
+                    "product-price": "...",
+                    "product-images": "...",
+                    "product-details": "...",
+                    "reviews-page": "...",
+                    "reviews-data": "...",
+                    "cta-text": 1,
+                    "shipping-text": 1
+                   },
+                   {
+                    "product-tag-id": "sunglasses",
+                    "brand-label": "...",
+                    "brand-favicon": "...",
+                    "product-title": "Spectacular Spectacles",
+                    "product-price": "...",
+                    "product-images": "...",
+                    "product-details": "...",
+                    "reviews-page": "...",
+                    "reviews-data": "...",
+                    "cta-text": 1,
+                    "shipping-text": 1
+                   },
+                   {
+                    "product-tag-id": "backpack",
+                    "brand-label": "...",
+                    "brand-favicon": "...",
+                    "product-title": "Beastly Backpack",
+                    "product-price": "...",
+                    "product-images": "...",
+                    "product-details": "...",
+                    "reviews-page": "...",
+                    "reviews-data": "...",
+                    "cta-text": 1,
+                    "shipping-text": 1
+                   }
+                ]
+              }
+          </script>
+        </amp-story-shopping-config>
         <amp-story-grid-layer template="vertical">
-          <amp-story-shopping-tag></amp-story-shopping-tag>
+          <amp-story-shopping-tag tag-data-id="hat" ></amp-story-shopping-tag>
+          <amp-story-shopping-tag tag-data-id="sunglasses" ></amp-story-shopping-tag>
+          <amp-story-shopping-tag tag-data-id="backpack" ></amp-story-shopping-tag>
         </amp-story-grid-layer>
         <amp-story-shopping-attachment layout='nodisplay'></amp-story-shopping-attachment>
       </amp-story-page>
-
-      <amp-story-page id="dark-theme">
-        <amp-story-shopping-config layout='nodisplay'></amp-story-shopping-config>
-        <amp-story-grid-layer template="vertical">
-          <amp-story-shopping-tag></amp-story-shopping-tag>
-        </amp-story-grid-layer>
-        <amp-story-shopping-attachment layout='nodisplay' theme="dark"></amp-story-shopping-attachment>
-      </amp-story-page>
```

Comment:
This page is an example of dark theme. We need to keep an example of dark=theme.

As a principal, example stories should contain a small example of each configuration option.